packages/frontend/src/components/Footer/index.tsx

```diff
@@ -42,7 +43,7 @@ export const Footer = () => {
         <Grid item mobile={12} tablet={6} desktop={6}>
           <Typography variant="body1" sx={{ marginTop: '24px' }}>
             Väyläviraston julkinen verkkopalvelu
-            <Link href="https://vayla.fi/" style={LinkStyle}>
+            <Link href="https://vayla.fi/" target="_blank" style={LinkStyle}>
```

Comment:
I'd add `rel="noopener"` when opening to new tabs, even if they are to Väylävirasto. Just good security practice.